<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>225</width>
    <height>201</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="myLineEdit1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Source</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="myLineEdit2">
          <property name="text">
           <string>Destination</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="myPushButton1">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="myPushButton2">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="myLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="incButton">
          <property name="text">
           <string>Increment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decButton">
          <property name="text">
           <string>Decrement</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Спискота</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>myPushButton2</sender>
   <signal>clicked()</signal>
   <receiver>myLineEdit2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myPushButton2</sender>
   <signal>clicked()</signal>
   <receiver>myLineEdit1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
